packages/contract-state-stream/test-app/client/src/utils/getWeb3.js

```diff
@@ -0,0 +1,38 @@
+import Web3 from "web3";
```

Comment:
I don't think we need this file. But then again, there are many files here we don't need.